tools/harness-thci/README.md

```diff
@@ -1,25 +1,26 @@
-OpenThread THCI
-===============
+THCI & Test Environment Setup
+=============================
 
-OpenThread THCI (Thread Host Controller Interface) is an implementation of the Python abstract class template "IThci",
+THCI (Thread Host Controller Interface) is an implementation of the Python abstract class template "IThci",
 which is used by Thread Test Harness Software to control OpenThread-based reference devices according to each test
 scenario.
 
-The current implementation is based on the CC2538 example platform and is included in the current Thread Test Harness
-Software release.
+Currently, there are two THCI implementations for OpenThread. One is for OpenThread CLI which is based on the CC2538 
+example platform and is included in the current Thread Test Harness Software release. The other is for OpenThread WpanCtl
+which is based on wpantund running on a linux host (For example: Raspberry Pi 3B) working with a NCP (Network Co-Processor) 
+board (For example: Nordic Semiconductor nRF52840) running a OpenThread NCP image.
 
 Platform developers should modify the THCI implementation directly to match their platform (e.g. serial baud rate).
 Alternatively, platform developers may follow the instructions below to add a new THCI implementation to the Test
-Harness. Adding a new THCI implementation allows the existing CC2538-based THCI implementation to coexist with the
-new THCI implementation.
+Harness.
 
-## Environment Setup ##
+## OpenThread Environment Setup ##
 
-1. Copy the "OpenThread.png" to /GRL/Thread1.1/MiniWeb/htdocs/images folder.
+1. Copy the "OpenThread.png" to \GRL\Thread1.1\Web\images folder.
 
-2. Copy the "deviceInputFields.xml" to /GRL/Thread1.1/MiniWeb/htdocs/data folder.
+2. Copy the "deviceInputFields.xml" to \GRL\Thread1.1\Web\data folder.
 
-3. Copy the "OpenThread.py" to /GRL/Thread1.1/Thread_Harness/THCI folder.
+3. Copy the "OpenThread.py" to \GRL\Thread1.1\Thread_Harness\THCI folder.
 
 4. Connect DUT(Device Under Test), sniffer and CC2538DK(or hardware running OpenThread) as reference device to Host PC.
```

Comment:
Make sure to have spaces between words and opening parentheses

"Connect DUT (Device Under Test), sniffer, and CC2538DK (or hardware running OpenThread) as reference devices to the Host PC."